src/components/charts/Vis3.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react'
+import { Scatter } from 'react-chartjs-2'
+import DATA from './annualMeanCO2.json'
+import DATA2 from './monthlyMeanCO2.json'
+import zoomPlugin from 'chartjs-plugin-zoom'
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js'
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  zoomPlugin
+)
+
+export const options = {
```

Comment:
Also would be nice to have some labels on axis's so it would explain what is x and y, and also specify units, like ℃, **ppm**, etc.